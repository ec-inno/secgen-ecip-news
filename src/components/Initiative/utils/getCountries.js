const getCountriesTranslated = t => {
  return [
    { value: 'bg', label: t('Bulgaria') },
    { value: 'es', label: t('Spain') },
    { value: 'cs', label: t('Czech Republic') },
    { value: 'da', label: t('Denmark') },
    { value: 'de', label: t('Germany') },
    { value: 'et', label: t('Estonia') },
    { value: 'el', label: t('Greece') },
    { value: 'fr', label: t('France') },
    { value: 'ga', label: t('Ireland') },
    { value: 'hr', label: t('Croatia') },
    { value: 'it', label: t('Italia') },
    { value: 'lv', label: t('Latvia') },
    { value: 'lt', label: t('Lithuania') },
    { value: 'hu', label: t('Hungary') },
    { value: 'mt', label: t('Malta') },
    { value: 'nl', label: t('Netherlands') },
    { value: 'pl', label: t('Poland') },
    { value: 'pt', label: t('Portugal') },
    { value: 'ro', label: t('Romania') },
    { value: 'sk', label: t('Slovakia') },
    { value: 'sl', label: t('Slovenia') },
    { value: 'fi', label: t('Finland') },
    { value: 'sv', label: t('Sweden') },
  ];
};

export default getCountriesTranslated;
