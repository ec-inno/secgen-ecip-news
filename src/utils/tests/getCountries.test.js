import getCountries from '../getCountries';

describe('getCountries utlity', () => {
  it('returns a list of values', () => {
    const t = s => s; // mocks https://www.i18next.com/overview/api#t
    const countries = getCountries(t);
    expect(countries).toMatchSnapshot();
  });
});
