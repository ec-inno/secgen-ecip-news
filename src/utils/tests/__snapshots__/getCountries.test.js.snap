// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`getCountries utlity returns a list of values 1`] = `
Object {
  "ad": "Andorra",
  "al": "Albania",
  "am": "Armenia",
  "at": "Austria",
  "be": "Belgium",
  "bg": "Bulgaria",
  "cy": "Cyprus",
  "cz": "Czech Republic",
  "da": "Denmark",
  "de": "Germany",
  "dk": "Denmark",
  "ee": "Estonia",
  "es": "Spain",
  "fi": "Finland",
  "fr": "France",
  "ga": "Ireland",
  "gb": "United Kingdom",
  "gr": "Greece",
  "hr": "Croatia",
  "hu": "Hungary",
  "ie": "Ireland",
  "it": "Italia",
  "lt": "Lithuania",
  "lu": "Luxembourg",
  "lv": "Latvia",
  "mt": "Malta",
  "nl": "Netherlands",
  "pl": "Poland",
  "pt": "Portugal",
  "ro": "Romania",
  "se": "Sweden",
  "si": "Slovenia",
  "sk": "Slovakia",
}
`;
